### PR TITLE
background image validation

### PR DESCRIPTION
**Pull request by mistermendez**
undefined live stream promo image on start screen was causing console error.

added image validation to all background images.

### CODE REVIEW

File: js/components/utils.js [hunk 1]
```diff
@@ -465,6 +466,29 @@ var Utils = {
...
 
+  /**
+   * Checks if image loads and is valid
+   *
+   * @function isValidImage
+   * @param {String} src - absolute path to image src
+   * @returns {Boolean} true if image is valid, false if not
+   */
+  isValidImage: function(src) {
+    if (src) {
```
Comment (dchauOoyala) [inline js/components/utils.js]: Would it be better here to check that `src` is a string and also not empty? Anything of truthy value will pass here (i.e. 1, a non-empty object, etc)

Comment (mistermendez) [inline js/components/utils.js]: good catch will update logic. also noticed return boolean isn't being updated by onerror function.

File: js/components/utils.js [hunk 2]
```diff
@@ -465,6 +466,21 @@ var Utils = {
...
 
+  /**
+   * Checks if string is valid
+   *
+   * @function isValidString
+   * @param {String} src - string to be validated
+   * @returns {Boolean} true if string is valid, false if not
+   */
+  isValidString: function(src) {
+    if (src && (typeof src === 'string' || src instanceof String)) {
```
Comment (dchauOoyala) [inline js/components/utils.js]: consider simplifying this to: `return (src && (typeof src === 'string' || src instanceof String))`

**Comment by mistermendez** [comment]
actual image validation (done via network request) would need to be done asynchronously and overkill for this issue. simplified logic to src string validation.

**Comment by dchauOoyala** [comment]
LGTM

**Pull request merged**